core/server/api/v2/utils/serializers/input/options.js

```diff
@@ -0,0 +1,43 @@
+const _ = require('lodash');
+
+const trimAndLowerCase = (params) => {
```

Comment:
Feels like this `trimAndLowerCase` should belong to some `/shared/utils` thing :thinking:

Reply:
Could be yes, but it's currently only used here. No priority IMO